src/project.jl

```diff
@@ -85,8 +92,8 @@ function project!(
                   workarray[thid].ifft_plan)
     end
 
-    # add up to get view
-    for y in 1:plan.imgsize[2]
+    copy3dj!(view, plan.add_img, 1) # initialize accumulator
+    for y in 2:plan.imgsize[2] # accumulate to get total view
```

Comment:
This is the key modification @ZongyuLi-umich 
Please see if you agree with it and the other changes in this PR.